README.md

```diff
@@ -17,3 +17,33 @@ The developer reward program for this project is taking place. [Check the detail
 ## How to contribute:
 
 Read the [contributing guide](https://github.com/dev-protocol/protocol/blob/master/.github/CONTRIBUTING.md), and create PR when you have time. 🧚✨
+
+## How to setup
+
+Executing the following command will compile each contract.
+
+```
+git clone https://github.com/dev-protocol/protocol.git
+cd protocol
+npm init
+npm install -g truffle
+truffle init
+  →Proceed anyway?　：Yes  Other question :No
+npm install -E openzeppelin-solidity
+truffle compile
```

Comment:
This part is can be more simple, following:
```bash
npm i
npm run generate
```

Reply:
If you want to use Truffle commands, you can [npx](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner) command, like the following:

```bash
npx truffle compile
```

This command is using locally installed Truffle instead of globally installed Truffle.